src/MessageTrait.php

```diff
@@ -30,24 +30,20 @@ trait MessageTrait
     /**
      * List of all registered headers, as key => array of values.
      *
-     * @var array
      * @psalm-var array<non-empty-string, list<string>>
      */
-    protected $headers = [];
+    protected array $headers = [];
 
     /**
      * Map of normalized header name to original name used to register header.
      *
-     * @var array
      * @psalm-var array<non-empty-string, non-empty-string>
      */
-    protected $headerNames = [];
+    protected array $headerNames = [];
```

Comment:
Adding types to protected properties is a BC break, also, removing `@var array` is fine - it would be good to rename the `@psalm-var` annotations to `@var` - the psalm prefix is unnecessary